climpred_skeleton/comparison.yaml

```diff
@@ -0,0 +1,11 @@
+EnsembleToObservation:
+  keywords: ['e2o', 'test']
+  long_name: 'verify ensemble mean against observations.'
```

Comment:
can this be modular? how would a user add his/her new comparison? would this always involve changing the yaml also?

Reply:
I didn't think about the user level of this. But if this is being done at the user level they don't need a keyword. They'd just enter `UserComparison` class into the `comparison=...` like we do currently.

The YAML is really easy on the developers to have a very readable list of metadata here. See future comment on how it will be used..